app/js/appRouter.js

```diff
@@ -8,7 +8,7 @@ define(function(require) {
 	var Marionette = require('marionette');
 
 	var routes = {
-		'gameWindow(/)': 'gameWindow',
+		'gameWindow(/)(:id)': 'gameWindow',
```

Comment:
Yeah, I know that, but dunno how to do better. First I did it like `gameWindow(/:id)`, but this way a request to **/gameWindow/** with a **/** at the end didn't work. So I put it as it is now. Another solution could be to make a separate `gameWindow/` route handler, but that's not really [DRY](https://ru.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself).